requirements.txt

```diff
@@ -5,12 +5,14 @@ click==6.6
 cryptography==1.4
 fauxfactory==2.0.9
 Inflector==2.0.11
+import_string==0.1.0
 mock==2.0.0
 paramiko==2.0.2
 pygal==2.2.3
 pylint==1.6.4
 pytest==2.9.2
 pytest_services==1.1.14
+redis==2.10.5
```

Comment:
Why adding redis as a requirement? For local runs we should be able to use file lock instead, so we should not insist on installing it. Imo it's a good candidate for requirements-optional.txt, but not requirements.txt